File: dotnet/cafdemalihapa/Directories/Repository/Dotnet/Cafdemalihapa/CafdemalihapaDirectory.cs
namespace cafdemalihapa.Directories.Repository.Dotnet.Cafdemalihapa
{
    public class CafdemalihapaDirectory
    (
        DotnetDirectory dotnetDirectory
    )
    {
        public string GetPath()
        {
            var dotnetDirectoryPath = dotnetDirectory.GetPath();
            var cafdemalihapaDirectoryPath = Path.Combine(dotnetDirectoryPath, "cafdemalihapa");
            return cafdemalihapaDirectoryPath;
        }
    }
}

File: dotnet/cafdemvers/zzq-cafdemwimu/cafdemwimu.console/Directories/Repository/Dotnet/Cafdemalihapa/CafdemalihapaDirectory.fs
namespace cafdemwimu.console.Directories.Repository.Dotnet.Cafdemalihapa

open System.IO
open cafdemwimu.console.Directories.Repository.Dotnet

type CafdemalihapaDirectory(dotnetDirectory: DotnetDirectory) =
    member _.GetPath() =
        let dotnetDirectoryPath = dotnetDirectory.GetPath()
        let cafdemalihapaDirectoryPath = Path.Combine(dotnetDirectoryPath, "cafdemalihapa")
        cafdemalihapaDirectoryPath
